client/src/components/Navbar/style.module.scss

```diff
@@ -7,13 +7,17 @@
 }
 
 .navbarActive {
-  color: green;
+  color: #008000;
 }
 
 .button {
   border: 0;
   padding: 0;
-  background-color: white;
+  background-color: #ffffff;
```

Comment:
Please, do `#fff`.

**Why?**
hex consists of 3 pairs
1 - `ff`
2 - `ff`
3 - `ff`
if in each pair we have 2 same digit, then we can shorten it by getting only 1 digit from each pair `#fff`.
https://stackoverflow.com/questions/10230331/how-to-convert-3-digit-html-hex-colors-to-6-digit-flex-hex-colors

Another topic: **why is it important to write as less code as we can?**
Because on the web we load all this code from server to the user's browser.
And the less code we have the faster our site will be loaded for our user.
And why it needs to be loaded as fast as possible I think you know very well 😃